cmd/cmd.go

```diff
@@ -67,6 +93,58 @@ var (
 	gaugeVersion    bool
 )
 
+func handleRepeatCommand(cmd *cobra.Command, cmdArgs []string) {
+	if repeat {
+		prevCmd := readPrevCmd()
+		executeCmd(cmd, prevCmd.Command)
+	} else {
+		if prevFailed {
+			prevFailed = false
+			return
+		}
+		if cmd.Name() == "run" {
```

Comment:
Can we move this to `PreRun` on the run command? 
This check for the command name explicitly is not needed then.